packages/cryptoplease/lib/features/popular_tokens/src/bl/bloc.dart

```diff
@@ -61,7 +61,7 @@ class PopularTokenBloc extends Bloc<PopularTokenEvent, PopularTokenState> {
       result.foldAsync(
         (e) => state.copyWith(processingState: ProcessingState.error(e)),
         (tokens) => state.copyWith(
-          tokens: tokens,
+          tokens: tokens.removeWhere((k, _) => k == Token.sol),
```

Comment:
The logic is now scattered across bloc and widgets.

Make this logic into a repository instead: let it fetch the data, and ensure that Solana is there (probably, by making another request).